src/test/kotlin/ar/edu/unahur/obj2/vendedores/VendedorTest.kt

```diff
@@ -35,5 +52,69 @@ class VendedorTest : DescribeSpec({
         viajante.puedeTrabajarEn(villaDolores).shouldBeFalse()
       }
     }
+
+    describe("esFirme"){
+      viajante.esFirme().shouldBeFalse()
+    }
+
+    describe("esInfluyente"){
+      viajante.esInfluyente().shouldBeFalse()
+    }
+  }
+
+  describe("comercioCorresponal") {
+    val hurlingham = Ciudad(buenosAires)
+    val avellaneda = Ciudad(buenosAires)
+    val caballito = Ciudad(buenosAires)
+    val palomar = Ciudad(buenosAires)
+    val haedo = Ciudad(buenosAires)
+    val comercio = ComercioCorresponsal(listOf(hurlingham, avellaneda, caballito, palomar, haedo))
+    val certificacion = Certificacion(esDeProducto = true, puntaje = 100)
+
+    comercio.agregarCertificacion(certificacion)
+
+    describe("puedeTrabajarEnCiudad"){
+      comercio.puedeTrabajarEn(hurlingham).shouldBeTrue()
+    }
+
+    describe("esFirme") {
+      comercio.esFirme().shouldBeTrue()
+    }
+
+    describe("esInfluyente") {
+      comercio.esInfluyente().shouldBeTrue()
+    }
+
   }
+
+    describe("centroDistribucion") {
+      val centro = CentroDistribucion()
+      val hurlingham = Ciudad(buenosAires)
+      val viajante = Viajante(listOf(misiones))
+      val vendedorFijo = VendedorFijo(hurlingham)
+
+
+
+
+      centro.agregarVendedor(viajante)
+      centro.agregarVendedor(vendedorFijo)
+      centro.agregarVendedor(vendedorFijo)
+
+      describe("esVendedorEstrella"){
+        centro.esVendedorEstrella() == viajante
```

Comment:
Esto no lo toma como un test, para que ejecute la comparación tenés que usar alguno de los métodos que empiezan con `should...`. En este caso sería así:

```suggestion
        centro.esVendedorEstrella().shouldBe(viajante)
```